integration-test/src/main/resources/application.yml

```diff
@@ -7,6 +7,14 @@ spring:
 logging:
     path: ${integrationtest.outputdir}
 
+cleanupapp:
```

Comment:
Do we need this? I mean we need to have the same account as the test run in, there is no real point defining it twice. It can only lead to errors.
Or at least it should default to the integrationtest variables

Reply:
`cleanupapp.user.accessKey` and `cleanupapp.user.secretKey` not that keys as we have in the `integrationtest` for credentials and not same as the integration test user's have. The environment can be same. So I'm checking this out how can it work if the `server` parameters are coming from the `integrationtest`.